data/shared/citizen/scripting/v8/timer.js

```diff
@@ -11,10 +11,10 @@
 
     let animationFrames = [];
 
-    function setTimer(timer, callback, interval) {
+    function setTimer(timer, callback, timeout) {
```

Comment:
Given how `setTimeout` is in fact just `setInterval` but will die after first run this should be kept as `interval`.